frontend/src/components/Header.vue

```diff
@@ -12,7 +12,7 @@
       </a>
 
       <v-toolbar-title>
-        <v-btn class="title hidden-sm-and-down" color="text" text>{{ appTitle }}</v-btn>
+        <v-btn class="title hidden-sm-and-down" color="text" text>Common Service Get Token</v-btn>
```

Comment:
We want this pulling from VUE_APP_TITLE, not hard coded. This allows us to control the title based on environment.

Reply:
totally agree. this was a temporary fix. i wondered if changing the environment variables would be done along with changes to server config map stuff in a different ticket ?

Reply:
VUE_APP environment variables work differently. Look at the appTitle field and can see it's pulling from process.env.VUE_APP_TITLE, then see the .env files and the VUE_APP_* vars are set in there.
These could be config map-pulled values but for now are managed by the vue app variables.
For our discussions of changing how GETOK FE is hosted (served by it's api, rather then static on Caddy), this paradigm might change anyways.

But for current implementation should be able to just change the .env file for the title.

Reply:
awesome. that's good to know. thanks